eve/main.yml

```diff
@@ -2042,7 +2055,7 @@ stages:
           name: Run installation scenarii on the bastion
           env:
             <<: *_env_bastion_tests
-            PYTEST_FILTERS: "install and ci and multinodes and not node2"
+            PYTEST_FILTERS: "install and ci and multinodes and not node$((%(prop:nodes_count)s + 1))"
```

Comment:
Yes but we do not need more than 2 nodes for the moment and since it's only internal I think it's ok but you right if one day we need more we will need to update the expansion "test" (or change the way we do expansion in the CI :wink: )